app/assets/javascripts/discourse/app/components/user-nav/messages-nav.hbs

```diff
@@ -9,64 +9,64 @@
 
   <ul class="messages-nav nav-pills action-list">
     {{#if @isPersonal}}
-      <li>
+      <li class="messages-latest">
         <LinkTo @route="userPrivateMessages.index" @model={{@user}}>
-          {{i18n "categories.latest"}}
+          {{d-icon "envelope"}}<span>{{i18n "categories.latest"}}</span>
```

Comment:
Maybe `span`s on their own line? Mostly so it matches the changes in `user-nav.hbs`.